lib/grim/image_magick_processor.rb

```diff
@@ -21,14 +21,30 @@ def count(path)
     end
 
     def save(pdf, index, path, options)
-      width   = options.fetch(:width,   Grim::WIDTH)
-      density = options.fetch(:density, Grim::DENSITY)
-      quality = options.fetch(:quality, Grim::QUALITY)
+      width      = options.fetch(:width,   Grim::WIDTH)
+      density    = options.fetch(:density, Grim::DENSITY)
+      quality    = options.fetch(:quality, Grim::QUALITY)
       colorspace = options.fetch(:colorspace, Grim::COLORSPACE)
-      command = [@imagemagick_path, "-resize", width.to_s, "-antialias", "-render",
-        "-quality", quality.to_s, "-colorspace", colorspace,
-        "-interlace", "none", "-density", density.to_s,
-        "#{Shellwords.shellescape(pdf.path)}[#{index}]", path]
+      alpha      = options[:alpha]
+
+      command = []
+      command << @imagemagick_path
+      command << "-resize"
+      command << width.to_s
+      command << "-alpha #{alpha}" if alpha
```

Comment:
I thought it was a good idea because it only needs one if .... I think I would prefer a variant where the other args are in one line too, eg:

    command << "-resize #{width.to_s}"

Your call,  I'm okay with both variants.